stories/features/prepare/main.fmf

```diff
@@ -0,0 +1,5 @@
+title: Package Preparation
+summary: Preparing packages for testing
+story:
+    As a user, I want a flexible and a comfortable way to prepare
```

Comment:
Nitpick: there probably shouldn't be `a` before `comfortable` (the article ties to the noun)

Reply:
Thanks, fixed.